docs/support/eject.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Moving away from Nitric
+description: How to move away from using the Nitric framework
+---
+
+## Freedom of choice
+
+In this section, we will discuss how you can move away from using the Nitric Framework. The Nitric team would love for you to continue using the framework, but only if it serves you.
+
+Our philosophy has always been freedom, choose your cloud, choose your language and use your preferred dev tools. To achieve this goal we've had to be opinionated about our technology decisions and realize that sometimes it might not fit your needs.
+
+## Open-source
+
+The Nitric framework is open-source and leverages other great open source technologies like Pulumi and Terraform. This means that you will always have full visibility over whats going on in the framework at both build and runtime.
+
+If a feature is missing, the first step is to get in touch:
+
+- [Discord](https://discord.gg/Webemece5C)
+- [Email](mailto:team@nitric.io)
+
+A great time to contact us is when your team requires support for a native resource which we do not yet support or you need to extend the default configuration of an existing resource.
+
+> A Nitric team member can help you build a provider with the IaC tool of choice to satisfy your requirements and add more value to the greater community.
+
+## Choosing a new IaC
+
+Most of the code that you write for your applications will likely be specific to your organization or domain. Nitric's role is to provide supporting runtime and deploy time resources which make it easier for you to build your app. We have eliminated the need for a separate deployment project, so this will be where the most effort lies in migration away.
```

Comment:
```suggestion
Most of the code that you write for your applications will likely be specific to your organization or domain. Nitric's role is to provide supporting runtime and deploy time resources which make it easier for you to build your app. We have eliminated the need for a separate deployment project, so this will be where the most effort lies in migrating away.
```